packages/sdk/src/chains/BaseAssetTransfer.ts

```diff
@@ -72,6 +73,26 @@ export abstract class BaseAssetTransfer {
     return transfer;
   }
 
+  /**
+   * Returns all registered resources between source domain of asset transfer and provided destination domain (on-chain check).
+   *
+   * @param {string} destinationDomainID - Destination domain ID
+   */
+  public async getRegisteredResourcesTo(destinationDomainID: string): Promise<Resource[]> {
+    const registeredResources: Resource[] = [];
```

Comment:
small nitt. Although only primitives are constants in js and object can be filled with properties, to me it looks weird to read an array that is going to be filled later declared with const 😄

Reply:
I would argue here that it is more appropriate to define it as const, as you will not change this reference.